etc/config/cuda.gpu.properties

```diff
@@ -1,4 +1,4 @@
 # Compiler settings inherited from amazon.properties to keep them in sync
-# Disabled: nvcc nvrtc
-compilers=&nvcc:&nvrtc
+# Disabled: nvcc
+compilers=&nvcc
```

Comment:
Why has the nvrtc group been removed here? Seems like that means we will no longer have them on the site?